test/e2e/scalenodes.go

```diff
@@ -43,7 +43,21 @@ var _ = Describe("Scale nodes", func() {
 			}
 			expectedNodeCount += int(*wp.Count)
 		}
-
+*/
+		
+		
+		// New One
+		machinesets, err := clients.MachineAPI.MachineV1beta1().MachineSets(machineSetsNamespace).List(ctx, metav1.ListOptions{})
+		if err != nil {
+			log.Warn(err)
```

Comment:
should we maybe error out here? otherwise the below code may also fail
```
		Expect(err).NotTo(HaveOccurred())
```

Reply:
Good catch!

Reply:
i changed that accordingly,